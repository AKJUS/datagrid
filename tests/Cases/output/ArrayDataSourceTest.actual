array(
	array(
		'id' => 1,
		'name' => 'John Doe',
		'age' => 30,
		'address' => 'Blue Village 1',
	),
	4 => array('id' => 30, 'name' => 'Jackie Blue', 'age' => 80, 'Prague 678'),
)
