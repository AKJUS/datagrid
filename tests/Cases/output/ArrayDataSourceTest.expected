array(array('id' => 30, 'name' => 'Jackie Blue', 'age' => 80, 'Prague 678'))
